planner/core/explain.go

```diff
@@ -186,8 +186,17 @@ func (p *PhysicalTableScan) AccessObject() string {
 // OperatorInfo implements dataAccesser interface.
 func (p *PhysicalTableScan) OperatorInfo(normalized bool) string {
 	buffer := bytes.NewBufferString("")
-	if p.pkCol != nil {
-		fmt.Fprintf(buffer, "pk col:%s, ", p.pkCol.ExplainInfo())
+	for i, pkCol := range p.PkCols {
+		var fmtStr string
+		switch i {
+		case 0:
+			fmtStr = "pk cols: {%s, "
+		case len(p.PkCols) - 1:
+			fmtStr = "%s}"
```

Comment:
```suggestion
			fmtStr = "pk cols: (%s, "
		case len(p.PkCols) - 1:
			fmtStr = "%s)"
```

Keep the same format for index columns in the explain info.